cogs/login.py

```diff
@@ -0,0 +1,150 @@
+import json
+import logging
+from pathlib import Path
+from typing import Any, Dict
+
+import discord
```

Comment:
its better to not import all of discord and import the things necessary to avoid stuff like
`discord.ButtonStyle.green`